code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1235,12 +1235,53 @@
 	default_container = /obj/effect/decal/cleanable/greenglow
 	/// How much tox damage to deal per tick
 	var/tox_damage = 0.5
+	/// How radioactive is this reagent
+	var/rad_power = 1
 
 /datum/reagent/uranium/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
 	. = ..()
+	if(!HAS_TRAIT(affected_mob, TRAIT_IRRADIATED) && SSradiation.can_irradiate_basic(affected_mob))
+		radiation_pulse(
+			source = affected_mob,
+			max_range = 0,
```

Comment:
I will note your PR says `This radiation pulse only affects the target and does not apply to surrounding areas`

...But with a max range of 0, won't it affect any mob in the mob's turf?

Reply:
Small edge case oversight. Technically when a mob becomes radioactive they pulse radiation naturally. Anyway I switched it to use `SPT_PROB` instead since we ignore any kind of rad protection from clothing and other sources.

Fixed via https://github.com/tgstation/tgstation/pull/89560/commits/3a0ed2c3a23c343c376640b93155c91855a04d57